Necrosis.lua

```diff
@@ -474,55 +498,52 @@ function Necrosis_OnUpdate()
 	if SpellTimer then
 		if update then
 			textTimersDisplay = "";
-			local GraphicalTimer = {texte = {}, TimeMax = {}, Time = {}, titre = {}, temps = {}, Gtimer = {}};
 			for index = 1, table.getn(SpellTimer), 1 do
 				if SpellTimer[index] then
 					if (GetTime() <= SpellTimer[index].TimeMax) then
+						clear_tables(GraphicalTimer);
```

Comment:
I'm not actually sure this clear call is necessary as the values get overridden in Necrosis_DisplayTimer anyway.